plugin-tester-java/src/main/java/example/myapp/helloworld/LoggingErrorHandlingGreeterClient.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * license agreements; and to You under the Apache License, version 2.0:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This file is part of the Apache Pekko project, which was derived from Akka.
+ */
+
+package example.myapp.helloworld;
+
+import example.myapp.helloworld.grpc.GreeterServiceClient;
+import example.myapp.helloworld.grpc.HelloReply;
+import example.myapp.helloworld.grpc.HelloRequest;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import org.apache.pekko.actor.ActorSystem;
+import org.apache.pekko.grpc.GrpcClientSettings;
+
+public class LoggingErrorHandlingGreeterClient {
+
+  public static void main(String[] args) {
+
+    // Boot pekko
+    ActorSystem sys = ActorSystem.create("LoggingErrorHandlingGreeterClient");
+
+    // Take details how to connect to the service from the config.
+    GrpcClientSettings clientSettings =
+        GrpcClientSettings.connectToServiceAt("127.0.0.1", 8082, sys).withTls(false);
+    // Create a client-side stub for the service
+    GreeterServiceClient client = GreeterServiceClient.create(clientSettings, sys);
+
+    try {
+      // #client-calls
```

Comment:
It looks like these markers were once used to include this snippet in the docs, but no longer. I think they can be removed.